### PR TITLE
Navigate to wallet assets after send

### PR DESCRIPTION
**Pull request by zaelgohary**
### Changes

- Fix format errs & warnings
- Navigate back on send

### Related Issues

- https://github.com/threefoldtech/threefold_connect/issues/802

### Tested Scenarios

- Send to another wallet

### CODE REVIEW

File: app/lib/screens/preference_screen.dart [hunk 1]
```diff
@@ -266,45 +266,6 @@ class _PreferenceScreenState extends ConsumerState<PreferenceScreen> {
     return await checkBiometricsAvailable();
   }
 
-  void _showDisableFingerprint() {
```
Comment (AhmedHanafy725) [inline app/lib/screens/preference_screen.dart:269]: why do you remove this method?

File: app/lib/widgets/wallets/send_confirmation.dart [hunk 2]
```diff
@@ -164,6 +164,8 @@ class _SendConfirmationWidgetState extends State<SendConfirmationWidget> {
         loading = false;
       });
       return;
+    } finally {
+      Navigator.pop(context);
```
Comment (AhmedHanafy725) [inline app/lib/widgets/wallets/send_confirmation.dart]: in the failure case, we shouldn't go back

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: app/lib/screens/preference_screen.dart (hunk 1 above)
Comment (zaelgohary) [reply app/lib/screens/preference_screen.dart:269]: It's not being used anywhere.

File: app/lib/widgets/wallets/send_confirmation.dart (hunk 2 above)
Comment (zaelgohary) [reply app/lib/widgets/wallets/send_confirmation.dart]: Fixed.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**